### PR TITLE
SQL-2103: Publish signing keys for JDBC

### PR DESCRIPTION
**Pull request by EthanHardyMongo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ MongoDB Atlas SQL JDBC driver requires Java 1.8 or higher.
 You can download the precompiled driver (jars) from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).  
 Choose `jar` in the Download dropdown for the lean jar (dependencies not included) or `all.jar` for the fat jar (dependencies bundled inside the jar). The name of the driver class is `com.mongodb.jdbc.MongoDriver`.
 
+### Verify Integrity of MongoDB JDBC Driver Packages
+To verify the integrity of the JDBC Driver packages, the artifact signature needs to be verified. To do this, there are two steps:
+1. Import the public key:  
+- You will need to import the key from one of the three following servers: `pgp.mit.edu`, `keys.openpgp.org`, `keyserver.ubuntu.com`.
```
Comment (jchemburkar) [inline README.md:22]: are we sure all 3 of these are correct? mit and ubunutu worked for me but openpgp gives "gpg: keyserver receive failed: No data"

File: README.md [hunk 2]
```diff
@@ -16,6 +16,17 @@ MongoDB Atlas SQL JDBC driver requires Java 1.8 or higher.
 You can download the precompiled driver (jars) from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).  
 Choose `jar` in the Download dropdown for the lean jar (dependencies not included) or `all.jar` for the fat jar (dependencies bundled inside the jar). The name of the driver class is `com.mongodb.jdbc.MongoDriver`.
 
+### Verify Integrity of MongoDB JDBC Driver Packages
+To verify the integrity of the JDBC Driver packages, the artifact signature needs to be verified. To do this, there are two steps:
+1. Import the public key:  
+- You will need to import the key from one of the three following servers: `pgp.mit.edu`, `keys.openpgp.org`, `keyserver.ubuntu.com`.  
+- To import the key, run the following command: `gpg --keyserver <server url> --recv-keys 30666110`.
```
Comment (jchemburkar) [inline README.md:23]: nit: we can probably merge this and the following example into a single bullet

**Review by jchemburkar — CHANGES_REQUESTED** [review]
Just a couple quick thoughts after testing, otherwise looks good

File: README.md (hunk 1 above)
Comment (nbagnard) [reply README.md:22]: This is so odd, Ethan had the same issue. It worked fine for me.

File: README.md [hunk 3]
```diff
@@ -16,6 +16,17 @@ MongoDB Atlas SQL JDBC driver requires Java 1.8 or higher.
 You can download the precompiled driver (jars) from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).  
 Choose `jar` in the Download dropdown for the lean jar (dependencies not included) or `all.jar` for the fat jar (dependencies bundled inside the jar). The name of the driver class is `com.mongodb.jdbc.MongoDriver`.
 
+### Verify Integrity of MongoDB JDBC Driver Packages
+To verify the integrity of the JDBC Driver packages, the artifact signature needs to be verified. To do this, there are two steps:
```
Comment (nbagnard) [inline README.md:20]:
```suggestion
To verify the integrity of the JDBC Driver packages, follow the steps below:
```

File: README.md [hunk 4]
```diff
@@ -16,6 +16,17 @@ MongoDB Atlas SQL JDBC driver requires Java 1.8 or higher.
...
+1. Import the public key:  
+- You will need to import the key from one of the three following servers: `pgp.mit.edu`, `keys.openpgp.org`, `keyserver.ubuntu.com`.  
+- To import the key, run the following command: `gpg --keyserver <server url> --recv-keys 30666110`.  
+- For example, `gpg --keyserver keys.openpgp.org --recv-keys 30666110`.
```
Comment (nbagnard) [inline README.md:24]:
```suggestion
The public key ID is 91A2157730666110.   

To import the key, run the following command: `gpg --keyserver <server url> --recv-keys 91A2157730666110`. 
Replace <server url> with one of the current GPG Keyservers supported by Maven Central Servers:
    - keyserver.ubuntu.com
    - keys.openpgp.org
    - pgp.mit.edu
```

File: README.md [hunk 5]
```diff
@@ -16,6 +16,17 @@ MongoDB Atlas SQL JDBC driver requires Java 1.8 or higher.
...
+### Verify Integrity of MongoDB JDBC Driver Packages
+To verify the integrity of the JDBC Driver packages, the artifact signature needs to be verified. To do this, there are two steps:
+1. Import the public key:  
+- You will need to import the key from one of the three following servers: `pgp.mit.edu`, `keys.openpgp.org`, `keyserver.ubuntu.com`.  
+- To import the key, run the following command: `gpg --keyserver <server url> --recv-keys 30666110`.  
+- For example, `gpg --keyserver keys.openpgp.org --recv-keys 30666110`. 
+2. Verify artifact:  
+- For this step, you will need to create a local copy of the `.jar.asc` file from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc) in addition to downloading the `.jar` file.  
+- To verify the signature, run the following command `gpg --verify mongodb-jdbc-<version>.jar.asc mongodb-jdbc-<version>.jar`  
+- For example, `gpg --verify mongodb-jdbc-2.1.2.jar.asc mongodb-jdbc-2.1.2.jar`.
```
Comment (nbagnard) [inline README.md:28]:
```suggestion
- Download the artifact you want to verify along with the matching detached signature `.asc` file, from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).
 For example, mongodb-jdbc-2.1.2.jar and mongodb-jdbc-2.1.2.jar.asc.
- To verify the signature, run the following command `gpg --verify <detached_signature_file> <artifact_to_verify>`.
 For example, `gpg --verify mongodb-jdbc-2.1.2.jar.asc mongodb-jdbc-2.1.2.jar`.
```

**Review by nbagnard — CHANGES_REQUESTED** [review]
This is looking good. But I agree with Jon that we could make things even shorter.
I left suggestions.

Make sure to provide the long key ID and not the short one (since the short one is apparently not supported by [keys.openpgp.org](http://keys.openpgp.org/)) if you don't apply the suggestions as is.

**Review by nbagnard — APPROVED** [review]
LGMT!

**Review by jchemburkar — APPROVED** [review]
lgtm!

**Pull request merged**